OpenTerm/Base.lproj/Main.storyboard

```diff
@@ -597,6 +634,9 @@ This project uses the open source ios_system project by Nicolas Holzschuch. This
         <namedColor name="Panel Background Color">
             <color white="0.064999997615814209" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
         </namedColor>
+        <namedColor name="Panel Background Color">
```

Comment:
This looks very close to the existing "Panel Background Color", but now we have multiple? Probably just Interface Builder doing floating point math...